components/layout_2020/dom_traversal.rs

```diff
@@ -448,10 +460,31 @@ where
                             vec.push(PseudoElementContentItem::Replaced(replaced_content));
                         }
                     },
+                    ContentItem::OpenQuote | ContentItem::CloseQuote => {
+                        // TODO(xiaochengh): calculate quote depth
+                        let depth: usize = 0;
```

Comment:
Please remove this and simply use the constant.

Reply:
Done